fe/fe-core/src/main/java/com/starrocks/load/loadv2/SparkLoadJob.java

```diff
@@ -475,70 +475,72 @@ private Set<Long> submitPushTasks() throws UserException {
-                                        // deep copy TBrokerScanRange because filePath and fileSize will be updated
-                                        // in different tablet push task
-                                        TBrokerScanRange tBrokerScanRange =
-                                                new TBrokerScanRange(params.tBrokerScanRange);
-                                        // update filePath fileSize
-                                        TBrokerRangeDesc tBrokerRangeDesc = tBrokerScanRange.getRanges().get(0);
-                                        tBrokerRangeDesc.setPath("");
-                                        tBrokerRangeDesc.setStart_offset(0);
-                                        tBrokerRangeDesc.setSize(0);
-                                        tBrokerRangeDesc.setFile_size(-1);
-                                        if (tabletMetaToFileInfo.containsKey(tabletMetaStr)) {
-                                            Pair<String, Long> fileInfo = tabletMetaToFileInfo.get(tabletMetaStr);
-                                            tBrokerRangeDesc.setPath(fileInfo.first);
-                                            tBrokerRangeDesc.setStart_offset(0);
-                                            tBrokerRangeDesc.setSize(fileInfo.second);
-                                            tBrokerRangeDesc.setFile_size(fileInfo.second);
-                                        }
-
-                                        // update broker address
-                                        Backend backend = GlobalStateMgr.getCurrentState().getCurrentSystemInfo()
-                                                .getBackend(backendId);
-                                        if (brokerDesc.hasBroker()) {
-                                            FsBroker fsBroker = GlobalStateMgr.getCurrentState().getBrokerMgr().getBroker(
-                                                    brokerDesc.getName(), backend.getHost());
-                                            tBrokerScanRange.getBroker_addresses().add(
-                                                new TNetworkAddress(fsBroker.ip, fsBroker.port));
-                                            LOG.debug("push task for replica {}, broker {}:{}, backendId {}," + 
-                                                    "filePath {}, fileSize {}", replicaId, fsBroker.ip, fsBroker.port, 
-                                                    backendId, tBrokerRangeDesc.path, tBrokerRangeDesc.file_size);
-                                        } else {
-                                            LOG.debug("push task for replica {}, backendId {}, filePath {}, fileSize {}",
-                                                    replicaId, backendId, tBrokerRangeDesc.path,
-                                                    tBrokerRangeDesc.file_size);
+                                PushBrokerReaderParams params = getPushBrokerReaderParams(table, indexId);
```

Comment:
What's the purpose of the pr? Feature/BugFix/Enhancement or Refactor. You have to describe it in the commit message.